### PR TITLE
Added support for serviceAccount (#68)

### LINKED ISSUE #68: Support serviceAccount when starting pipeline/task

### PR DESCRIPTION
**Pull request by lstocchi**
it resolves #68

### CODE REVIEW

**Comment by lstocchi** [comment]
@jeffmaury added combo with serviceAccounts taken from the cluster

File: src/main/java/com/redhat/devtools/intellij/tektoncd/utils/StartResourceModel.java [hunk 1]
```diff
@@ -147,7 +176,7 @@ private void setDefaultValueResources() {
             result.addAll(getInputsFromNodeInternal(resources, Input.Kind.RESOURCE));
         }
 
-        return result.isEmpty() ? null : result;
+        return result.isEmpty() ? Collections.emptyList() : result;
```
Comment (jeffmaury) [inline src/main/java/com/redhat/devtools/intellij/tektoncd/utils/StartResourceModel.java:179]: Return result directly

File: src/main/java/com/redhat/devtools/intellij/tektoncd/utils/StartResourceModel.java [hunk 2]
```diff
@@ -170,7 +199,7 @@ private void setDefaultValueResources() {
             }
         }
 
-        return result.isEmpty() ? null : result;
+        return result.isEmpty() ? Collections.emptyList() : result;
```
Comment (jeffmaury) [inline src/main/java/com/redhat/devtools/intellij/tektoncd/utils/StartResourceModel.java:202]: Return result directly

**Comment by jeffmaury** [comment]
As you updated the model constructor, you must update the corresponding test and see if there are new test methods to be added as well

**Comment by lstocchi** [comment]
> As you updated the model constructor, you must update the corresponding test and see if there are new test methods to be added as well

Yes i added a couple of tests to verify if serviceName/taskServiceNames are correctly filled in when using a pipeline vs a task. Let me know if you think we need something else

**Pull request merged**